soft4pes/control/lin/state_space_curr_ctr.py

```diff
@@ -0,0 +1,254 @@
+""" State-space current controller with anti-windup scheme for grid-connected converter with RL load """
+
+from types import SimpleNamespace
+import numpy as np
+from soft4pes.utils.conversions import alpha_beta_2_dq, dq_2_abc, dq_2_alpha_beta
+
+
+class RLGridStateSpaceCurrCtr:
+    """
+    State-space current controller with anti-windup scheme for grid-connected converter with RL load.
+    
+    Attributes
+    ----------
+    Rf : float
+        Resistance [p.u.].
+    Xf : float
+        Reactance [p.u.].       
+    base : base-value object
+        Base values.
+    Ts : float
+        Sampling interval [s].
+    Ts_pu : float
+        Sampling interval [p.u.].
+    uc_ii_dq : 1 x 2 ndarray of floats
+        Converter voltage reference after integral gain block in dq frame [p.u.].
+    uc_ref_dq : 1 x 2 ndarray of floats
+        Converter voltage reference after saturation block in dq frame [p.u.].
+    uc_uc_ref_dq_unsat : 1 x 2 ndarray of floats
+        Converter voltage reference before saturation block in dq frame [p.u.].
+    uc_km1 : 1 x 2 ndarray of floats
+        Pervious measurment of converter voltage in dq frame [p.u.].                                      
+    i_ref_seq_dq : Sequence
+        Current reference sequence instance in dq-frame [p.u.].   
+    sim_data : dict
+        Controller data.
+    """
+
+    def __init__(self, sys, base, Ts, i_ref_seq_dq):
+        """
+        Initialize a RLGridStateSpaceCurrCtr.
+
+        Parameters
+        ----------
+        sys : system object
+           System model.
+        base : base-value object
+           Base values.
+        Ts : float
+           Sampling interval [s].
+        i_ref_seq_dq : Sequence
+           Current reference sequence instance in dq-frame [p.u.].
+        """
+
+        self.Xf = sys.Xg  # Consider the filter inductane equals to the grid inductance
```

Comment:
In my opinion, for future development, we need to consider the filter. After applying the filter in our model system, we should change just sys.Xg to sys.Xf. So, this description if you want can change. I have done all the variables and parameter names based on the paper.

Reply:
The filter will be added in the future, but we will make a new controller for that (or modify this to be more general and work with both systems, but this will be addressed then). 

I know I required all the variables to be named as in the paper, but now looking at this line, ` self.Xf = sys.Xg`, it is clearly implied that the filter reactance is actually the grid reactance. For me, it would be more clear if we did not imply with the variables that we have a filter when we do not have that. 

If you feel that the code is cleaner with self.Xf, you may use that.